server/apps/immich/src/api-v1/auth/dto/sign-up.dto.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { plainToInstance } from 'class-transformer';
```

Comment:
Have we included this file in the test suite? I haven't seen it run in the test action yet.

Reply:
Yes `class-transformer` is in the project. Basically this unit test is replicating what the framework does to hydrate the DTOs from the requests.
Since the validation is not inside the service, I don't think there other way to unit-test it 🙂